src/frontend/src/pages/HomePage/components/SubtaskSection.tsx

```diff
@@ -0,0 +1,83 @@
+import { Typography, useTheme, Grid } from '@mui/material';
+import Checkbox from '@mui/material/Checkbox';
+import { Box } from '@mui/system';
+import React from 'react';
+import { Checklist } from 'shared';
+
+const SubtaskSection: React.FC<{ subtasks: Checklist[]; parentTask: Checklist }> = ({ subtasks, parentTask }) => {
+  const theme = useTheme();
+
+  return (
+    <Box
+      sx={{ backgroundColor: '#CECECE', padding: 2, marginTop: -0.5, marginBottom: 3, borderRadius: '0px 0px 10px 10px' }}
+    >
+      {subtasks.length > 0 ? (
+        <Grid container sx={{ display: 'flex', alignContent: 'center', justifyContent: 'center', alignItems: 'center' }}>
+          <Grid item xs={12} md={7}>
+            {subtasks.map((subtask) => (
+              <Box marginLeft={5} display={'flex'} alignItems={'center'} marginBottom={1}>
+                <Checkbox
+                  sx={{
+                    '& .MuiSvgIcon-root': {
+                      fill: 'black',
+                      backgroundColor: 'black',
+                      borderRadius: 1
+                    },
+                    '&.Mui-checked .MuiSvgIcon-root': {
+                      backgroundColor: 'white'
+                    },
+                    '&:hover': {
+                      backgroundColor: 'transparent'
+                    }
+                  }}
+                />
+                <Typography color={'black'} fontWeight={'bold'}>
+                  {subtask.name}
+                </Typography>
+              </Box>
+            ))}
+          </Grid>
+          <Grid
+            item
+            xs={12}
+            md={5}
+            sx={{
+              backgroundColor: theme.palette.background.paper,
+              padding: 2,
+              borderRadius: 2
+            }}
+          >
+            <Typography color={theme.palette.common.white}>{parentTask.descriptions[0]}</Typography>
+          </Grid>
+        </Grid>
+      ) : (
+        <Box
```

Comment:
wait so we can only show descriptions when there aren’t subtasks, i thought in some designs there was one description off to the right side and then the bullets were still shown on the left. could be mistaken.

Reply:
Yeah so there are descriptions for if there are subtasks but they are pushed to the right side because of the subtasks, this is so that if there are no subtasks, the description is in the middle

Reply:
Becayse this allows for multiple descriptions if there are no subtasks but if you notice above if there are subtasks, we only want to display the parent checklists description